### ISSUE TITLE
CLI output broken in CI/CD

### ISSUE THREAD

**Issue by aaronweatherall** [postmanlabs/newman#556]
Hey guys, great work on the new release BUT..

```
build	24-Aug-2016 16:35:28	??? PA - Reuse refresh token
build	24-Aug-2016 16:35:28	  POST http://<url_removed>[200 OK, 305B, 42ms]
build	24-Aug-2016 16:35:29	  ???
build	24-Aug-2016 16:35:29	  ??? null
build	24-Aug-2016 16:35:29	  ???
build	24-Aug-2016 16:35:29	  ???  Has expected status code (200,201)
build	24-Aug-2016 16:35:29	  ???  Has content-type
build	24-Aug-2016 16:35:29	  ???  Has expected content type (application/json)
build	24-Aug-2016 16:35:29	  ???  Has valid schema
build	24-Aug-2016 16:35:29	  ???  Has access token in body
build	24-Aug-2016 16:35:29	
build	24-Aug-2016 16:35:29	??? PA - Update client id
build	24-Aug-2016 16:35:29	  POST http://<url_removed> [200 OK, 305B, 31ms]
build	24-Aug-2016 16:35:29	  ???
build	24-Aug-2016 16:35:29	  ??? null
build	24-Aug-2016 16:35:29	  ???
build	24-Aug-2016 16:35:29	  ???  Has expected status code (200,201)
build	24-Aug-2016 16:35:29	  ???  Has content-type
build	24-Aug-2016 16:35:29	  ???  Has expected content type (application/json)
build	24-Aug-2016 16:35:29	  ???  Has valid schema
build	24-Aug-2016 16:35:29	  ???  Has access token in body
build	24-Aug-2016 16:35:29	
build	24-Aug-2016 16:35:29	??? PA - Update client id (Invalid)
build	24-Aug-2016 16:35:29	  POST http://<url_removed> [400 Bad Request, 335B, 30ms]
build	24-Aug-2016 16:35:29	  ???
build	24-Aug-2016 16:35:29	  ??? null
build	24-Aug-2016 16:35:29	  ???
build	24-Aug-2016 16:35:29	  ???  Has expected status code (400)
build	24-Aug-2016 16:35:29	  ???  Has content-type
build	24-Aug-2016 16:35:29	  ???  Has expected content type (application/problem+json)
build	24-Aug-2016 16:35:29	  ???  Has valid schema
build	24-Aug-2016 16:35:30	
```

The cool characters and colors are nice BUT they're totally broken on bamboo and out other CI/CD tools.. where's the option to *not* use funky characters??

I've tried running with ` run /etc/newman/collection.json -e /etc/newman/environment.json --no-color --reporter-junit-export="junit/junit.xml`

But still get the same results

**Comment by shamasis** [comment]
I see that the colours have actually turned off automatically. That's because otherwise it would have looked funkier!!! Even the spinners automatically turned off. 

What did not get turned off is the tick marks. I am wondering why bamboo is not rendering those since they're standard UTF code page characters? They even work on Windows!

Tried Newman on travis and Shippable. They work fine as well. 

I will work on replacing the tick marks with pass fail texts. But it would be great if you could check why bamboo isn't rendering standard UTF character set. 😞

**Comment by shamasis** [comment]
@aaronweatherall also would you be able to test out a beta release and let us know? Would then experiment with different variants of tick marks!! 😜

**Comment by aaronweatherall** [comment]
Yeah we're running your docker container, so should be easy enough

**Comment by shamasis** [comment]
Also @aaronweatherall can you verify this - https://confluence.atlassian.com/bamkb/how-to-change-the-wrong-encoding-in-bamboo-that-is-causing-the-application-to-crash-318670134.html ?

@elssar is the locale and code page of our docker set to UTF8?

**Comment by aaronweatherall** [comment]
Actually that's a good point, there's no default encoding in your dockerfile for alpine.

```ENV LC_ALL=en_US.UTF-8 \
    LANG=en_US.UTF-8 \
    LANGUAGE=en_US.UTF-8
```

I'm about to try it with the utf8 env variables

**Comment by shamasis** [comment]
@aaronweatherall - were you able to fix the UTF8 thing in docker? @elssar can you release the latest newman beta as an image?

**Comment by mykhaylo-** [comment]
I've faced similar issue when invoking newman from Jenkins. But this doesn't seem to be an issue with newman. As I understand it just generates output in utf-8, but CI/CD use different encoding.
After applying the fix from https://issues.jenkins-ci.org/browse/JENKINS-31096 (basically just need to add `-Dfile.encoding=UTF-8`) I've get rid of this issue.

**Comment by shamasis** [comment]
@mykhaylo- thanks.
@aaronweatherall - were you able to fix this by setting the encoding

**Comment by kunagpal** [comment]
@aaronweatherall @mykhaylo- We've added a fix in #597, it will become available in the next release.

**Comment by kunagpal** [comment]
@aaronweatherall @mykhaylo-  This bug should have been fixed in Newman `v3.1.1`, please check it out and get back to us if your issue persists. Thanks!
